_1327/main/views.py

```diff
@@ -8,6 +10,9 @@
 from _1327.documents.models import Document
 from _1327.documents.utils import permission_warning
 
+from .forms import MenuItemForm
```

Comment:
We actually do this in a lot of places if both modules are in the same Django app, so I think it's ok.